From 7c83b8d5a0c7a93035b7d762a3396388d342a868 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Tue, 30 May 2017 18:32:39 -0700
Subject: [PATCH 3/5] Fix Nuspec

Since we don't currently ILMerge, we need to fix the nuspec so that we
don't expect to pick up an ILMerged version of NuGet
---
 .../NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.nuspec         | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.nuspec b/src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.nuspec
index bee38e9..89ef833 100644
--- a/src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.nuspec
+++ b/src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.nuspec
@@ -8,7 +8,6 @@
     <description>NuGet 3 pack for dotnet CLI</description>
   </metadata>
   <files>
-    <file src="..\..\..\artifacts\NuGet.Build.Tasks.Pack.Library\15.0\bin\$configuration$\net45\*.dll" target="Desktop\" />
     <file src="..\..\..\artifacts\NuGet.Build.Tasks.Pack.Library\15.0\bin\$configuration$\netstandard1.3\*.dll" target="CoreCLR\" />
     <file src="Pack.targets" target="buildCrossTargeting\NuGet.Build.Tasks.Pack.targets" />
     <file src="Pack.targets" target="build\NuGet.Build.Tasks.Pack.targets" />
-- 
1.8.3.1

